releasenotes/notes/sql-obfuscator-conditional-compute-cache-key-a7b53b18597373f7.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improved the behavior of the SQL obfuscator cache key computation.
```

Comment:
```suggestion
    Improve the behavior of the SQL obfuscator cache key computation.
```

Reply:
since this is a backport I'm a bit worried of changing the wording in the stable branch but not in main, I'd rather merge it as it, sorry